pandas/tests/indexes/multi/test_constructor.py

```diff
@@ -463,3 +463,4 @@ def test_tuples_with_name_string():
         pd.Index(li, name='abc')
     with pytest.raises(ValueError):
         pd.Index(li, name='a')
+
```

Comment:
Can you undo the change in this file?

(it's extraneous to your PR)

Reply:
@gfyoung (unless I'm wrong) this comment ended up showing here by mistake?